modules/ROOT/Edits/configure-third-party.adoc

```diff
@@ -0,0 +1,21 @@
+= Configure and Manage 3rd Party Connections
+
+If your trading partners use 3rd party service providers such as VANs or 3PLs (3rd party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, you can create a 3rd party connection to manage configurations such as the endpoints, identifiers, and certificates of the 3rd party provider.
```

Comment:
```suggestion
If your trading partners use third-party service providers such as VANs or 3PLs (third-party logistics providers) who manage the B2B connectivity with your organization on behalf of your partners, you can create a third-party connection to manage configurations such as the endpoints, identifiers, and certificates of the third-party provider.
```

Reply:
At another company I worked for, there was discussion on whether we should even use the term "third party," it being a changeable relationship. I'm not sure "outside" or "independent" are any better, however; maybe you have ideas about this?

Reply:
GLOBALLY search for "3rd party" and replace; I will stop marking it up now. Hyphenate when used as an adjective but not as a noun.